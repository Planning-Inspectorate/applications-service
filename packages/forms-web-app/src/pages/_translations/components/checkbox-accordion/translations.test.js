const enTranslations = require('./en.json');
const cyTranslations = require('./cy.json');

describe('pages/_translations/components/checkbox-accordion/translations', () => {
	it('should return the correct English translations', () => {
		expect(enTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh translations', () => {
		expect(cyTranslations).toMatchSnapshot();
	});

	it('should return the same keys for English and Welsh translations', () => {
		expect(enTranslations).toHaveSameKeys(cyTranslations);
	});
});
