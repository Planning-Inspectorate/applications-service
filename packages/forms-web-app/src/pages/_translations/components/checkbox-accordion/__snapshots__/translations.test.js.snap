// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/_translations/components/checkbox-accordion/translations should return the correct English translations 1`] = `
{
  "accessibility": {
    "filterOptions": {
      "segmentText1": "Select to",
      "segmentText2": "the",
      "segmentText3": "filter options",
    },
  },
}
`;

exports[`pages/_translations/components/checkbox-accordion/translations should return the correct Welsh translations 1`] = `
{
  "accessibility": {
    "filterOptions": {
      "segmentText1": "Dewiswch i",
      "segmentText2": "yr",
      "segmentText3": "opsiynau hidlo",
    },
  },
}
`;
